t/ui/27-plugin_obs_rsync_obs_status.t

```diff
@@ -15,6 +15,16 @@ use Mojo::Server::Daemon;
 use Mojo::IOLoop::Server;
 use Mojo::IOLoop::ReadWriteProcess 'process';
 use Mojo::IOLoop::ReadWriteProcess::Session 'session';
+use Test::MockModule;
+
+my $mocked_time = 0;
+
+BEGIN {
+    *CORE::GLOBAL::time = sub {
+        return $mocked_time if $mocked_time;
+        return time();
+    };
+}
```

Comment:
Mocking this manually is totally fine (if it works). I'd just like to note that there's also the Perl module `Test::MockTime` which might come handy as well.

Reply:
https://metacpan.org/pod/Test::Mock::Time - that one we are already using in os-autoinst

Reply:
According to `ci-packages.txt` we use `Test::Mock::Time` which doesn't allow for going back in time: https://github.com/powerman/perl-Test-Mock-Time/blob/master/lib/Test/Mock/Time.pm#L63C6-L64C66

`Test::MockTime` (without the division) does in fact allow for selecting an arbitrary point in time.

Reply:
If you go 100 years into the future it won't be our problem anymore when the test fails in the future. 😉

Reply:
I've not actually looked at the code of `Test::Mock::Time`, but it seems rather event loop specific and might really not be usable here.

Reply:
Maybe I have confused the module with what we use in os-autoinst. I guess if you don't see a benefit over manual mocking then it isn't worth changing your code anyways.